src/app.js

```diff
@@ -136,13 +136,15 @@ const App = Marionette.Application.extend({
 
     this.setRegion(opts.region);
 
-    this.triggerMethod('before:start', options);
-
     opts.state = this.getInitState(opts.state);
```

Comment:
Since this will be a breaking change I think we can go ahead and remove `getInitState` and making `opts`